packages/flutter_tools/test/general.shard/compile_expression_test.dart

```diff
@@ -74,9 +75,10 @@ void main() {
     )));
 
     await generator.recompile(
-      '/path/to/main.dart',
+      Uri.parse('/path/to/main.dart'),
```

Comment:
Should this mimic the implementation by doing `globals.fs.file(...).uri` ?

Reply:
Done